corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -17,6 +17,7 @@
     window.feature_previews.ENUM_IMAGE = {{ request|feature_preview_enabled:"ENUM_IMAGE"|BOOL }};
     window.feature_previews.CALC_XPATHS = {{ request|feature_preview_enabled:"CALC_XPATHS"|BOOL }};
     window.toggles.MM_CASE_PROPERTIES = {{ request|toggle_enabled:"MM_CASE_PROPERTIES"|BOOL }};
+    window.toggles.CASE_LIST_DISTANCE_SORT = {{ request|toggle_enabled:"CASE_LIST_DISTANCE_SORT"|BOOL }};
```

Comment:
it'd be amazing to have this in `base.html`. (just thinking out load, don't think that's at all in scope)